assets/xplorers-bot-gcloud.drawio

```diff
@@ -0,0 +1 @@
3fLX7KwF5yS6gh8Q8kYd7wG+TM2XOUcUSATlyqsgpLSkHdJHplKP6nKf+3Dvm3x+pcGy9XDBvGQ6OFa7K83EQL/Ujz641GhKz3/NXo2GJ9sY9ADE2UC0YBeQB/QkgHNhaLGm5pGADQBjVZTropBuiqUwBZSwBkJKhDQDheFYOY6qrsM7iAU3h/jyjG7eAewVUkRzsBDXfcoEPVJr5F6rgsaIiD7D/1KWDjAAfCVCYA9ly5q8jpqC8Fu9gdeuDHP7yKIeq/EKOvgi2ZJfVA2s9vyxFjPpveTcQ+at3g/vuzbuByN8CZEv79lcD8VPk6qVeYP+DXX8/7BoIE58Wdd+8KWhpZcyXOTa7wK4cc5HqsJci4ydg14BjvvoeduWsgPz2K34S9alAFSNj+tylOuSUP0AeWfVZznOmG8UZLnUt4VlajeYuYEu2hC7HHr2Idv8BUIEeMa7urnvH7popqsgWBjnrrhHf8hO+Gby2eA6Zf4Vv24/xra9nh0aWr/eU0IcPoKezACXWFN5AXLBEoS2wpxQ5QBfoVgcunYk5oASvYd/tz7FXBuRCxOhV0oC1jxr9Tm9sQF1QXcQoVj7G1Y4FgR1laaIPAF83ycQ34DtNGWAu9M8ZGxsE7mPH6da2JN7RVl/v/kELiztq3Zehs3VzG/CehhnE1gY67WnKtgZ02hAXfIafBYU3jOqtJSSApYOykjHMkBdfWUS3KVqJeXKFv2AtkNXIoWMH/QyUH+IMaWplQQb9vw7xBTHVxNkwButLGnbh/hJ4VvKgJwTgiZ4SbH+aasoS3kimWJeRJ2HjCPSq4uZBPcXFqVMuMS6W2ye3AvNvwvN2QnacmHG3sx6h8x/E887/YCj9jQPo8+Ez+fnZ6Pl2WPyNk1npi5PZDxz85VeRf8td0heGn1dl6tNi+2lN/cKI67b0vV8d4XqdJHieJKMh+KFlb/mfnhVt3zyIqf1PC3bkh1W95IdIj7FeRpZnBnehY3vmHc93wQpDiUNTcn/NAhDsr2og6dh/QWW94/juz93W/YZsveO4q/ZL6tyka6f7zmS2+4f8f/+O/4dP2kzHLDg4zXT/Ih5A/eQyDG56pA8HlHsn9kqcpDZOvZ01YnLHp+H45eR9G22Ry5sXBNekd3qwDw99bg95vsG31421dAvED+9MHoQ/NXN/+D+Y2eTTzN6toxj+93cYI3eHKMDRyLBKs+E2DVfO/s7y9lbg3EHc1Iva9P63Twq/BSA+j6L35ld/LIq4r5yO/X8OoySL/n5hJF41hd3/7Si6PQB8a07hUK99ZDlx7MRtzUERtnbVA8W7aGvH1Q9HMOiwNuPdHTyz23tm4gTFPwXqy9hyOZ4WyTthwf9H4+J2bi2DY95MKzkfXTuuF7fb/TMD+lMzoFGwqd7PQ0ukPs6EloG1ZbsVL+KsJ8V5wXvnp2wmclRXC3jPzjVFTpk+EHGurI7WgbmwI7s6EXKziT+H+yq3WhDgbe+W4+foaaYWVJnjMwX118lqdnY25qr9eoqg9uXucvQSWkVvhxOL6nvpFqx/mjRUa4oescJhavFL0I7gBLswFlK5Gg3Jcubi9ICnKHGpipM+4ZgHb/S+lVPf2KlezRvWSstXNT9OlgrZRd7NhBotlKjKgMAbf0kVWlIvAw1U0IDyDCcIvpEyn8IebjzRB2TiW4Qqm1IdG+5Ex7OwQa7pU1EdZXjNT3yjxAn9RMG1Lq8pBqyN3NVMRnlFWk5FbZZlap8UONmnpSxo/Syh+jRl5TTTdOranpzglJzqA4kprrvqy+drc+TDgDcteh11RHBfnNwJTLcqO9BCLJmi8uzsOVZc7vHi4QnCHE8QSlb04nN5YO07e+CEhJJr2VlpXdplaLhUx8kPcrYuZMe1tOJjAG/qOl4vqXihvOVc0BQ3BruDTNOC+dMz21d6oUwCU2QJZOLxOW0m5lTfEK1/JT/46p190B9gFxeev9Zh86lvz2SRrn2rz+SCVicXEEelAXu7oP+UA1u39hAhW+CacuoAeOku8DZEqlvlRNlAzEDMKhuRznoL/PVCPTV1M+r1zMa/p6xUrGKiyI3/XIjTuYCTLh3tjGfK5ZTXdCOGf/D1AHzt5rUMm3q6q8iFOowgY5vfHMweD5ALvvn6LGn+4DhJBe/xjAde9eQ0WQEgG3xAHD3KNYFBXj7sl/ouXfFScLyecYK5eCamEgE6QPwpc5EKPfHsfmwupMDgh5kWPnL2eHpvCc/Fik+CySLfrcKXf634INVCbmePXoqz585lbJGtmkTiKdtEh9zyLrDDVZWzc/iQgP4UT5Qk6qsc+I+vT2eooM0w14wUcl6CmGviYA4YAnGgYD7LBexBmvgjYMeGruaQH00sTHPMKYgzhqda4D9OUwYc5lEVh/oUPucu0NBfuJZM6tMvAn4uaOVLC+65OeRwDHEF8siipuMMeVDipBL8SJiXVad+OBPGKeWijmvALBHjD+JxfpE/NR5V8pesxRhAZpT1pDP9UDc2y070GsNExFjQR6C4F2KUIoPMG5dVU9+pWOMB4pEF36d8k0s4XYbcmko1Tk5xGkwgloWJgmtdsJdavJdLGk6M9XmBcY6/iaimvZBLgEPneI+YWYCdQJ85+HiAuENw6gv5jniGGAM2myLGN36aF5iTkEMl6ImnHSn4gqOl617UFfgEnCGaDrHjVfkpVie8OmBF0aM3fCr744nIHP3Dqok3+AD8CxU1E4E3x3QV/i2X4bRan2cQhxCTc8QBqDkDwCXcByusBRiKeuA+F3wucn/hqW5zQgGx/pAZC7azx5sO1Y+/S+KM0uWWo3md3/2LfDrmoBZKh1U4P+Xka13VNR/84Rugs1taY/c6Z33jVe4Y2xdiK0lpjx8PJj9PVqMgXS52B3MhdlaLgEANPnvuTEY8XVpAjm6rEw3w3ybRrvqACW99dIL6Ydf9Df210Lm/fHdvZ+yfHK0K97/dXcPl6bfn9W9ITr/gFwb/DQ==</diagram></mxfile>
```

Comment:
something the image below doesn't show is how the function connects to the artifact registry and cloud run.